CharlieBackend.Api.UnitTes/DashboardServiceTest.cs

```diff
@@ -32,12 +54,80 @@ public async Task GetStudentsClassbook()
                 }
             };
 
-            var studentclassbookWrongRequest = new StudentsRequestDto<ClassbookResultType>()
+            var expectedStudentsMarks = new List<StudentMarkDto>
+            {
+                new StudentMarkDto
+                        {
+                            CourseId = 2,
+                            StudentGroupId = 1,
+                            StudentId = 5,
+                            LessonId = 2,
```

Comment:
Since these numeric values occur more than one time inside one test perhaps we could improve readability even more by declaring them as local variables
```cs
long expectedCourseId = 2;
long expectedStudentGroupId = 1;
// etc
```

Reply:
Good comment. Reworked in that commit https://github.com/ita-social-projects/WhatBackend/pull/409/commits/bb21b02db3e5a6bea2c2f9b538a67acea13efb34